src/hotspot/share/interpreter/interpreterRuntime.cpp

```diff
@@ -146,30 +146,30 @@ void InterpreterRuntime::set_bcp_and_mdp(address bcp, JavaThread *thread) {
 // Constants
 
 
-JRT_ENTRY(void, InterpreterRuntime::ldc(JavaThread* thread, bool wide))
+JRT_ENTRY(void, InterpreterRuntime::ldc(JavaThread* current /* TRAPS */, bool wide))
```

Comment:
I think the fact that these functions are JRT_ENTRY essentially means that they can throw an exception with TRAPS.  Adding this comment is just visual noise and in my case, confusion.  It doesn't help anyone.  I had to look back and see that ldc never had a TRAPS parameter, that's how obvious it was without this comment.